cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -678,8 +746,9 @@ void ImageFlip::disparityCallback(const stereo_msgs::DisparityImage::ConstPtr& d
 		rot_mat = cv::getRotationMatrix2D(center, -rotation_angle, 1.0);
 		if (switch_aspect_ratio==true)
 		{
-			rot_mat.at<double>(0,2) += 0.5*(disparity_image_turned.cols-disparity_image.cols);
```

Comment:
Just leave it the old way in this case, because the rotation matrix is always the same format CV_64FC1.